packages/sanity/src/core/i18n/bundles/studio.ts

```diff
@@ -56,6 +56,12 @@ export const studioLocaleStrings = {
   /* Accessibility label for navigating the calendar to "today", without _selecting_ today */
   'calendar.action.go-to-today-aria-label': 'Go to today',
 
+  /* Label for navigating the calendar to "yesterday", without _selecting_ yesterday. Short form, eg `Yesterday`, not `Go to yesterday` */
+  'calendar.action.go-to-previous-day': 'Yesterday',
```

Comment:
I think this (and the identifiers used in code) should use "yesterday" as part of the key instead of "previous", since "previous" requires context. Previous compared to what? If you have selected an arbitrary day in the calendar, "previous day" would be the day before the selected one.